modern-business/style-rtl.css

```diff
@@ -3415,59 +3415,77 @@ footer .wp-block-a8c-navigation-menu .sub-menu {
 /* Blocks */
 /* !Block styles */
 .entry .entry-content > *,
-.entry .entry-summary > * {
+.entry .entry-summary > *
+.fse-enabled .site-header > *,
+.fse-enabled .site-footer > * {
   margin: 32px 0;
   max-width: 100%;
 }
 
 @media only screen and (min-width: 768px) {
   .entry .entry-content > *,
-  .entry .entry-summary > * {
+  .entry .entry-summary > *
+.fse-enabled .site-header > *,
+  .fse-enabled .site-footer > * {
     max-width: calc(8 * (100vw / 12) - 28px);
   }
 }
 
 @media only screen and (min-width: 1168px) {
   .entry .entry-content > *,
-  .entry .entry-summary > * {
+  .entry .entry-summary > *
+.fse-enabled .site-header > *,
+  .fse-enabled .site-footer > * {
     max-width: calc(6 * (100vw / 12) - 28px);
   }
 }
 
 @media only screen and (min-width: 768px) {
   .entry .entry-content > *,
-  .entry .entry-summary > * {
+  .entry .entry-summary > *
+.fse-enabled .site-header > *,
+  .fse-enabled .site-footer > * {
     margin: 32px auto;
   }
 }
 
 .entry .entry-content > * > *:first-child,
-.entry .entry-summary > * > *:first-child {
+.entry .entry-summary > *
```

Comment:
It looks like there are a _ton_ of changes in these files - just wanted to make sure they're expected :)

Reply:
Nice catch, thanks - missing comma!

Reply:
Just good luck haha! I didn't even realize that